src/archiveThread.js

```diff
@@ -0,0 +1,26 @@
+/*jslint node: true */
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(mergeWithDefaults, api, ctx) {
+  return function archiveThread(thread_id, callback) {
+
+    var form = mergeWithDefaults();
+    form['ids['+thread_id+']'] = true;
```

Comment:
Would this work with more than one thread? If so we might as well support an array of `thread_id`s too.